v5/core/vpc_instance_authenticator.go

```diff
@@ -280,11 +280,6 @@ func (authenticator *VpcInstanceAuthenticator) invokeRequestTokenData() error {
 // and then (2) exchange that for an IAM access token.
 func (authenticator *VpcInstanceAuthenticator) RequestToken() (iamTokenResponse *IamTokenServerResponse, err error) {
 
-	// Use the default VPC base endpoint if user didn't specifiy the URL property.
```

Comment:
This check is actually not needed because of the way we initialize and use the URL field (through the url() method) within the `retrieveInstanceIdentityToken()` and `retrieveIamAccessToken()` methods.